python/tests/transformers/keras_transformer_test.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import numpy as np
+import os
+
+from keras.models import Sequential
+from keras.layers import Dense, Activation
+from keras.initializers import glorot_uniform
+
+from sparkdl.transformers.keras_transformer import KerasTransformer
+from ..tests import SparkDLTestCase
+
+class KerasTransformerTest(SparkDLTestCase):
+
+    # TODO(sid): How to create static class variables in Python? Should I just put these in the
```

Comment:
How should I handle these constants? Put them in a constructor, make them `@properties`, leave them as is?